streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/StickyTaskAssignor.java

```diff
@@ -36,6 +36,7 @@
     private final Map<TaskId, ID> previousActiveTaskAssignment = new HashMap<>();
     private final Map<TaskId, Set<ID>> previousStandbyTaskAssignment = new HashMap<>();
     private final TaskPairs taskPairs;
+    private boolean isStandByAssignment = false;
```

Comment:
not sure why we need this? We already pass a boolean through from `assignActive()` and `assignStandby()` to `allocateTaskWithClientCandidates(..)` why not just use that?

Reply:
ack